packages/prerender-proxy/index.ts

```diff
@@ -2,5 +2,6 @@ import { PrerenderLambda } from "./lib/prerender-lambda-construct";
 import { PrerenderFunction } from "./lib/prerender-construct";
 import { PrerenderCheckFunction } from "./lib/prerender-check-construct";
 import { ErrorResponseFunction } from "./lib/error-response-construct";
+import { CloudFrontCacheControl } from "./lib/prerender-cf-cache-control-consctruct"
```

Comment:
Type in file name: `./lib/prerender-cf-cache-control-consctruct` should be `./lib/prerender-cf-cache-control-construct`